setup.py

```diff
@@ -43,4 +43,8 @@
     ],
     license="GPLv3+",
     python_requires=">=3.6",
+    setup_requires=['setuptools-git-versioning'],
+    version_config={
+        "dev_template": "{tag}.rc{ccount}-git.{sha}",
```

Comment:
What do you think about using `post` (the default) instead of `rc`? Rc stands for release candidate and normally, `sprint-N.rc` should come *before* `sprint-N`. Post would make more sense to me.

Reply:
It does make more sence to me, too.